package.json

```diff
@@ -22,6 +22,7 @@
     "css"
   ],
   "devDependencies": {
+    "minimatch": "^9.0.1",
```

Comment:
minimatch is okay, but looks like it should be added to dependencies, not devDependencies.